apps/dh/api-dh/source/DataHub.WebApi/Program.cs

```diff
@@ -84,7 +84,7 @@
     .AddGraphQLServices()
     .ModifyRequestOptions(options =>
     {
-        options.IncludeExceptionDetails = environment.IsDevelopment();
+        options.IncludeExceptionDetails = true;
```

Comment:
suggestion: was it this that should not have been committed 😁

```suggestion
        options.IncludeExceptionDetails = environment.IsDevelopment();;
```

Reply:
Yes, fixed.